inception-project/inception
# #3887 - Allow selecting a concept by browsing the KB

**What's in the PR**
- Added browser button to simple concept feature editor
- Browsing is only enabled if the feature is bound to concept, instance or any (properties are not supported) and if there is either only one KB in the project or if the feature is bound to a specific KB (since there is no way to switch between KBs in the browser or to browse cross KB).

**How to test manually**
* Configure a concept feature to a specific KB and allowing any value
* Annotate a document and in particular use the browser button in the concept feature editor
* If the concept editor already contains a value, the path to that value should be expanded - if it is an instance, the instance should be highlighed

**Automatic testing**

**Documentation**
